### ISSUE TITLE
Schema with _id: false still has id

### ISSUE THREAD

**Issue by royibernthal** [TypedProject/tsed#746]
I'm using tsed 5.34.0.

I have a model which contains a schema.
I specified { schemaOptions: { _id: false } } in the Schema decorator, so the schema shouldn't have an id.
However, I'm still seeing an _id field inside the sub-document in the database.

Any idea why?

To be fair, I'm not even sure where I got schemaOptions from, I can't find it anywhere in the docs now. Is it perhaps an obsolete way to do it?

```
@Schema({ schemaOptions: { _id: false } })
export default class MySchema {

  @Property()
  some_property: string;

}

@Model()
export default class ModelWithSchema {

  @ObjectID('id')
  _id: string;

  @Property()
  my_schema: MySchema;

}
```

**Comment by Romakita** [comment]
Hello @royibernthal 

I checked the code  and I don't know where come from the problem. The options is correctly given to the Schema constructor. Maybe it's the id param the problem. I'll continue to investigate.

**Comment by royibernthal** [comment]
Hey @Romakita 

Okay thanks. For what it's worth, it used to work a while ago.

**Comment by Romakita** [comment]
Damned.... I’ll check the commits xD

**Comment by royibernthal** [comment]
Sounds like the way to go :) Were you able to repro?

**Comment by Romakita** [comment]
Hello @royibernthal 
I tried to reproduce your issue without success.
Here a PR where I added some unit test to be sure that the schemaOptions is correctly given the the Schema.

https://github.com/TypedProject/tsed/pull/752

You'll see the unit test confirm that is work as expected.
You can copy the unit test and test your model with your actual implementation. Maybe something is different in your side.

See you

**Comment by royibernthal** [comment]
Hey @Romakita 

Sorry for the late reply.

I tested this, admittedly not via a unit test, I just logged the schema's options.

I have 4 schemas that specify `@Schema({ schemaOptions: { _id: false } })` in exactly the same way.

3 out of those have the problem, while 1 somehow doesn't.

Those 3 problematic schemas are nested inside a different model than the 1 schema that works as expected - it might be a clue to what's going on.

The problem can be seen both in the database and in the logged schema options (which I assume is the heart of your unit test).
